### PR TITLE
1.23.0-0.2.1: [fix] Terms of service

### PR DESCRIPTION
**Pull request by taylorjdawson**
*  Fixes termsAgreed getting set on init w/o config being set
  * Adds classnames for customization of tos agreement

### CODE REVIEW

File: src/elements/Button.svelte [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <script lang="ts">
   import { app } from '../stores'
   export let onclick: () => void = () => {}
-  export let position: string
+  export let position: string = ''
```
Comment (taylorjdawson) [inline src/elements/Button.svelte:4]: Fixed a TS error

File: src/views/WalletCheck.svelte [hunk 2]
```diff
@@ -347,7 +347,7 @@
         </Button>
       {/if}
       {#if errorMsg}
-        <Button position={!activeModal.button && 'left'} onclick={doAction}>
+        <Button position={!activeModal.button ? 'left' : ''} onclick={doAction}>
```
Comment (taylorjdawson) [inline src/views/WalletCheck.svelte:350]: Fixes a TS error

File: src/views/WalletSelect.svelte [hunk 3]
```diff
@@ -325,7 +326,7 @@
         {selectedWalletModule}
         onBack={() => {
           selectedWalletModule = null
-          walletAlreadyInstalled = null
+          walletAlreadyInstalled = undefined
```
Comment (taylorjdawson) [inline src/views/WalletSelect.svelte:329]: Fixes a TS error

**Review by taylorjdawson — COMMENTED** [review]
Reviewed recent changes 👀

**Review by aaronbarnardsound — APPROVED** [review]

**Pull request merged**